core/db/migrate/20150113002122_create_spree_promotion_codes.rb

```diff
@@ -0,0 +1,13 @@
+class CreateSpreePromotionCodes < ActiveRecord::Migration
+  def change
+    create_table :spree_promotion_codes do |t|
+      t.references :promotion, index: true
+      t.string :value
+      t.datetime :starts_at
```

Comment:
The use case that I had in mind were:
- promotions that reoccur, like 'summer sales', maybe there's a 'summer14' and a 'summer15' with different dates
- temporary 'personalized' codes where a code generated for an individual would only be valid for a certain period of time